packages/sdks-tests/src/tests/basic-styles.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { expect } from '@playwright/test';
+import { test, __dirname, isRNSDK } from './helpers/index.js';
+import fs from 'fs';
+import path from 'path';
+
+test.describe('Basic styles', () => {
+  test.fail(isRNSDK);
+  test('styles should be applied consistently across all SDKs (text and buttons)', async ({
+    page,
+    packageName,
+  }) => {
+    test.skip(packageName === 'react-native');
+    await page.goto('/basic-styles');
+
+    const button = page.getByRole('button', { name: 'Click Me!' });
+
+    const buttonWidth = await button.evaluate(el => el.getBoundingClientRect().width);
+    expect(buttonWidth).toBe(1264);
+
+    const centeredAlignedText = page.locator('text=Centered aligned');
+    const leftAlignedText = page.locator('text=Left aligned');
+    const rightAlignedText = page.locator('text=Right aligned');
+
+    expect(await centeredAlignedText.evaluate(el => getComputedStyle(el).color)).toBe(
+      'rgb(0, 255, 16)'
+    );
+    expect(
+      await centeredAlignedText.evaluate(el => el.getBoundingClientRect().left)
+    ).toBeGreaterThan(580);
+
+    expect(await leftAlignedText.evaluate(el => getComputedStyle(el).color)).toBe('rgb(255, 0, 0)');
+    expect(await leftAlignedText.evaluate(el => el.getBoundingClientRect().left)).toBeLessThan(10);
+
+    expect(await rightAlignedText.evaluate(el => getComputedStyle(el).color)).toBe(
+      'rgb(0, 24, 255)'
+    );
+    expect(await rightAlignedText.evaluate(el => el.getBoundingClientRect().right)).toBeGreaterThan(
+      1200
+    );
+  });
+
+  test('Box with flex-direction row should have children aligned horizontally', async ({
+    page,
+  }) => {
+    const mockImgPath = path.join(__dirname, '..', '..', 'mocks', 'placeholder-img.png');
+    const mockImgBuffer = fs.readFileSync(mockImgPath);
+    await page.goto('/basic-styles');
+
+    const builderBox = await page.waitForSelector(
+      '[builder-id="builder-258a89336ec44f37a38a146e698a858a"]',
+      { timeout: 10000 }
+    );
```

Comment:
why is this timeout needed? You generally want to avoid arbitrary timeout durations in integration tests, it will cause all sorts of brittleness.

Why do you need to wait for the selector at all? Playwright is meant to take care of waiting until things are ready on its own.

```suggestion
    const builderBox = await page.selector(
      '[builder-id="builder-258a89336ec44f37a38a146e698a858a"]'
    )
```

Reply:
@samijaber should we then use `page.locator(...).elementHandle()`?

Reply:
this also doesn't fail correctly and leads to timeout, may need to pass `{ timeout: xxx }` to this

Reply:
wow! thanks for this